libraries/AP_Camera/AP_Camera.h

```diff
@@ -7,6 +7,7 @@
 
 #define AP_CAMERA_TRIGGER_TYPE_SERVO                0
```

Comment:
Not your problem to fix, but this should definitely be an `enum class` in the future.